static/js/lib/api/embedly.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import "isomorphic-fetch"
```

Comment:
Can we move this into `fetch_auth` or do we need this in each file?

Reply:
it seems we don't need it! tests pass and everything seems to work ok without. I'll remove it from all the API modules.